src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1756,17 +1787,17 @@ public class OptionsBase : TrainerInputBaseWithWeight
             /// <value>
             /// Set to 1 to simulate online learning.
             /// </value>
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum number of iterations; set to 1 to simulate online learning.", ShortName = "iter")]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum number of iterations; set to 1 to simulate online learning.", ShortName = "iter, MaxIterations")]
             [TGUI(Label = "Max number of iterations", SuggestedSweeps = "1,5,10,20")]
             [TlcModule.SweepableDiscreteParam("MaxIterations", new object[] { 1, 5, 10, 20 })]
-            public int MaxIterations = Defaults.MaxIterations;
+            public int NumberOfIterations = Defaults.NumberOfIterations;
 
             /// <summary>
             /// The initial <a href="tmpurl_lr">learning rate</a> used by SGD.
             /// </summary>
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Initial learning rate (only used by SGD)", ShortName = "ilr,lr")]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Initial learning rate (only used by SGD)", ShortName = "ilr,lr, InitLearningRate")]
             [TGUI(Label = "Initial Learning Rate (for SGD)")]
-            public double InitLearningRate = Defaults.InitLearningRate;
+            public double InitialLearningRate = Defaults.InitialLearningRate;
```

Comment:
>InitialLearningRate [](start = 26, length = 19)

So it works this way: in the beginning of each iteration we set learning rate to this one, and then with going through examples we decrease it. But in each iteration it starts with this one. 
So technically it's initial learning rate, but it remain same for each iteration....
Can you look other example of InitialLearningRate, and are they have same pattern or not?
I would probably just call it LearningRate.

Reply:
I do not see other examples of InitialLearningRate in the other learners.  

To me the names makes sense, esp. since it does indicate that  this learning rate is adaptive in nature. We can add more documentation as to why this helps .

---
In reply to: [261897003](https://github.com/dotnet/machinelearning/pull/2825#discussion_r261897003) [](ancestors = 261897003)